plugins/hydra_ray_launcher/examples/upload_download/conf/hydra/launcher/custom_ray_aws.yaml

```diff
@@ -9,12 +9,11 @@ sync_up:
   # we leave target_dir to be null
   # as a result the files will be synced to a temp dir on remote cluster.
   # the temp dir will be cleaned up after the jobs are done.
-  # recommend to leave target_dir to be null if you are syncing code/artifacts to remote cluster so you don't need
+  # recommend to leave target_dir to be null if you are syncing code/artifacts
+  # to remote cluster so you don't need
   # configure $PYTHONPATH on remote cluster
-  include: ['model', '*.py']
   # No need to sync up config files.
-  exclude: ['*']
+  exclude: ['conf', 'multirun', "**/__pycache__"]
 sync_down:
-  include: ['*.pt', '*/']
   # No need to sync down config files.
-  exclude: ['*']
+  exclude: ['conf']
```

Comment:
same here, conf won't always be the name of the conf dir. 
I guess for the example purpose it's OK, but i still think it's better to specify what you want download instead.